### PR TITLE
Adds FAQ

### LINKED ISSUE #6: Add the FAQ section
As noted in the README already, we could add an FAQ section, initialised with a question of whether you should choose A/B testing or MVT testing. 

I'm currently doing research into what the use cases for each of these kinds of testing are, and will then update this readme with this information.

### PR DESCRIPTION
**Pull request by ZooeyMiller**
Adds an FAQ section with information on factors that may effect your decision between going for an A/B test or an MVT test.
fix #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,12 +71,36 @@ You should _continually assess your priorities_. These are **suggested steps** b
...
 + also for interpreting your results section: http://www.optimizeandprophesize.com/jonathan_mendezs_blog/2008/05/multivariate-te.html
 + and https://econsultancy.com/blog/6740-what-your-mother-never-taught-you-about-multivariate-testing/
 
-### Possible FAQ
-+ Should I choose A/B or MVT?
-
-
+### FAQ
+#### Should I choose A/B or MVT?
+There are a few factors in whether to choose A/B or MVT testing, the main one is how long you have for the test to run before you need meaningful results.
```
Comment (iteles) [inline README.md:81]: @ZooeyMiller Is this not covered in https://github.com/iteles/learn-ab-and-multivariate-testing#multivariate-vs-ab-testing ? I wonder if we're better off containing this in one place and enhancing that (pointing one repo to the other) to reduce the maintenance burden of content changes?

**Review by iteles — COMMENTED** [review]
A question for you @ZooeyMiller !

File: README.md (hunk 1 above)
Comment (ZooeyMiller) [reply README.md:81]: I agree. The table in the learn-ab-... repo is a good explanation. Do you still think this explanation is valuable? If so I could PR to learn-ab... if not I'm fine with throwing it away!

Comment (iteles) [reply README.md:81]: Yes! I think fleshing out that section in learn-ab and then referring to it from here is valuable 😊

Comment (ZooeyMiller) [reply README.md:81]: Awesome, will close this PR and move info over to there then.

**Pull request closed without merging**